crates/flipperzero/src/dialogs/mod.rs

```diff
@@ -25,6 +25,7 @@ pub struct DialogMessage<'a> {
 }
 
 /// A dialog file browser options.
+#[repr(transparent)]
```

Comment:
Why is this necessary?

Reply:
While it is not used explicitly here, it allows tricks like pointer transmutations, for example between `*const DialogFileBrowserOptions` and `*const sys::DialogsFileBrowserOptions` which may come in handy in some cases.